rvt/default.py

```diff
@@ -316,6 +318,7 @@ def __init__(self):
         self.sim_bytscl = ("percent", 0.25, 0.00)
         self.ld_bytscl = ("value", 0.50, 1.80)
         self.msrm_bytscl = ("value", -2.50, 2.50)
+        self.mstp_bytscl = ("value", 0, 1)
```

Comment:
self.mstp_bytscl = ("value", 0.00, 1.00), for consistency and to write nicer json file entry (when saving default to json)